pages/_slug/index.vue

```diff
@@ -10,10 +10,10 @@
       <div
         v-for="section in page.section"
         :key="section.sys.id"
-        class="banner"
-      >
+        class="banner">
```

Comment:
when running via 'npm run dev' this line throws a linting error, it expects an additional line break.